components/FormikFields/PaperInputPicker/index.js

```diff
@@ -202,31 +208,48 @@ const PaperInputPicker = ({
               <View key={result.value}>
                 {/* selected value */}
                 {result.value === values[formikKey] && (
-                  <View>
-                    <Button
-                      style={layout.buttonGroupButtonStyle}
-                      key={result.value}
-                      mode="contained"
-                      onPress={() => setFieldValue(formikKey, result.value)}
+                  <TouchableWithoutFeedback OnPress={() => setFieldValue(formikKey, result.value)}>
+                    <View
+                      style={{
+                        backgroundColor: theme.colors.primary,
+                        borderColor: theme.colors.primary,
+                        borderWidth: 1,
+                        borderRadius: 5,
+                        alignItems: 'center',
+                        color: theme.colors.primary,
+                        paddingHorizontal: 15,
+                        paddingVertical: 5,
+                        margin: 5
```

Comment:
see if you can create a "button" styles object in the` index.styles.js` of this directory and put this code in there